doc/smart-switch/smart-switch-database-architecture/smart-switch-database-design.md

```diff
@@ -0,0 +1,262 @@
+# Smart Switch Database design #
+
+## Table of Content 
+
+- [Smart Switch Database design](#smart-switch-database-design)
+  - [Table of Content](#table-of-content)
+    - [Revision](#revision)
+    - [Scope](#scope)
+    - [Definitions/Abbreviations](#definitionsabbreviations)
+    - [Overview](#overview)
+    - [Requirements](#requirements)
+    - [Architecture Design](#architecture-design)
+      - [Database services](#database-services)
+      - [Database flow](#database-flow)
+    - [High-Level Design](#high-level-design)
+    - [SAI API](#sai-api)
+    - [Configuration and management](#configuration-and-management)
+      - [CLI/YANG model Enhancements](#cliyang-model-enhancements)
+      - [Config DB Enhancements](#config-db-enhancements)
+    - [Warmboot and Fastboot Design Impact](#warmboot-and-fastboot-design-impact)
+    - [Memory Consumption](#memory-consumption)
+      - [DPU\_APPL\_DB](#dpu_appl_db)
+      - [DPU\_APPL\_STATE\_DB/DPU\_STATE\_DB](#dpu_appl_state_dbdpu_state_db)
+    - [Restrictions/Limitations](#restrictionslimitations)
+    - [Testing Requirements/Design](#testing-requirementsdesign)
+      - [Unit Test cases](#unit-test-cases)
+      - [System Test cases](#system-test-cases)
+    - [Open/Action items - if any](#openaction-items---if-any)
+
+### Revision
```

Comment:
"##" will be better than "###"

Reply:
same as all sections below.

Reply:
This is just follow the hld template: https://github.com/sonic-net/SONiC/blob/master/doc/hld_template.md
I assume other docs also follow this convention.

Reply:
sorry, i am still a bit paranoid on this one... i think we should use "##" instead of "###", as "###" is currently messing up with navigation and side bar.